src/MenuItem/index.js

```diff
@@ -1,9 +1,9 @@
 import React from 'react'
 import PropTypes from 'prop-types'
-import Menu from './index'
 import cx from 'classnames'
 import styles from './styles'
 
+import Menu from '../Menu'
```

Comment:
Of course! Good catch. I'll have to open up the PropTypes to accept a react element instead of `instanceOf(Menu)` as that's the only thing I used it for here.

Reply:
cfe2950